doc/vldb23/relational.tex

```diff
@@ -624,4 +623,35 @@ \subsection{Complexity of incremental circuits}\label{sec:complexity}
 data persistently.  The space complexity of operators such as $\inc{(\lift{\distinct})}$,
 $\inc{(\lift{\bowtie})}$, $\I$, and $\D$ is $O(|DB[t]|)$.
 
+\subsubsection{IVM query plans and optimality}
+
+We want to spell explcitly what Algorithm~\ref{algorithm-inc} actually
+peforms.
+
+In relational algebra a query can be implemented by multiple plans,
+each with a different data-dependent cost\footnote{I.e., which plan is
+optimal may depend not only on the query, but also on the data.}.  The
+input of Algorithm~\ref{algorithm-inc} is not an abstract query, it is
```

Comment:
I would remove the "not an abstract query" part:

The input of Algorithm X is a non-incremental query plan produced by a query planner.